src/Symfony/Bridge/PhpUnit/TextUI/TestRunner.php

```diff
@@ -27,9 +27,16 @@ class TestRunner extends \PHPUnit_TextUI_TestRunner
      */
     protected function handleConfiguration(array &$arguments)
     {
+        $listener = new SymfonyTestsListener();
+
+        $result = parent::handleConfiguration($arguments);
+
         $arguments['listeners'] = isset($arguments['listeners']) ? $arguments['listeners'] : array();
-        $arguments['listeners'][] = new SymfonyTestsListener();
 
-        return parent::handleConfiguration($arguments);
+        if (!array_filter($arguments['listeners'], function ($listener) { return $listener instanceof SymfonyTestsListener; })) {
```

Comment:
you're right - yet this is a non issue in practice, isn't it? this array is usually almost empty